### PR TITLE
Fixes 126 MatlabID -> ID

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

File: R/utilities-goodness-of-fit.R [hunk 1]
```diff
@@ -25,9 +25,9 @@ plotMeanGoodnessOfFit <- function(structureSet,
     observations$metaData <- readObservedDataFile(structureSet$simulationSet$observedMetaDataFile)
     observations$filter <- structureSet$simulationSet$dataFilter
 
-    # TO DO: Agree on dictionary to replace "matlabID"
-    timeColumn <- as.character(observations$metaData[observations$metaData[, "matlabID"] == "time", "nonmenColumn"])
-    dvColumn <- as.character(observations$metaData[observations$metaData[, "matlabID"] == "dv", "nonmenColumn"])
+    # TO DO: Agree on dictionary to replace "ID"
```
Comment (Yuri05) [inline R/utilities-goodness-of-fit.R:28]: is it still a TODO or can we remove the comment?

File: R/utilities-goodness-of-fit.R [hunk 2]
```diff
@@ -497,9 +497,9 @@ plotPopulationGoodnessOfFit <- function(structureSet,
...
```
Comment (Yuri05) [inline R/utilities-goodness-of-fit.R:500]: same here

File: R/utilities-goodness-of-fit.R (hunk 1 above)
Comment (pchelle) [reply R/utilities-goodness-of-fit.R:28]: My bad, I forgot to remove them

**Comment by Yuri05** [comment]
Closing (implemented as part of another PR, s. https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/pull/178#issuecomment-623996689)

**Pull request closed without merging**